covasim/webapp/cova_app.css

```diff
@@ -32,6 +48,18 @@ html, body {
     margin: 1em 0;
 }
 
+.sidebar {
+    background-color: #F8F9FA !important;
+}
+
+.parheading {
```

Comment:
Would be better to override the card-header class.

Reply:
Added to issue